src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java

```diff
@@ -76,6 +76,11 @@ public class AddClubPopUpStepOne extends AddClubPopUpContainer {
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-placeholder']")
     private WebElement selectPlaceholder;
 
+    @FindBy(xpath = ".//span[text()='Приналежність до центру']/following-sibling::div")
+    @Getter(AccessLevel.NONE) private WebElement centerDropdown;
+
+    private DropdownElement centerDropdownElement;
+
```

Comment:
```suggestion
    @FindBy(xpath = ".//span[text()='Приналежність до центру']/following-sibling::div")
    @Getter(AccessLevel.NONE) private WebElement centerDropdown;

    private DropdownElement centerDropdownElement;

```

U have xpath for it  73, 74 lines

Reply:
Xpath "./descendant::span[@class='ant-select-selection-placeholder']" doesn't work if center is already chosen.